docs/pages.jl

```diff
@@ -17,7 +17,7 @@ pages = Any["Home" => "index.md",
                                            "catalyst_applications/nonlinear_solve.md",
                                            "catalyst_applications/bifurcation_diagrams.md"],
             "Inverse Problems" => Any["inverse_problems/optimization_ode_param_fitting.md",
-                                        "inverse_problems/petab_ode_param_fitting.md",
+                                        #"inverse_problems/petab_ode_param_fitting.md",
```

Comment:
PEtab is not MTK v9 updated yet, so commented out here (permitting docs to build).